### DISCUSSION TITLE
Optimized mode not automatically switching to Eco mode on startup on 2022 G14

### DISCUSSION THREAD

**Discussion by xzvs** [seerge/g-helper#3251] [Q&A]
When I shut off my laptop in Optimized mode plugged into AC power, it's in iGPU + dGPU mode. But after I unplug it and start it up again, (fast startup is disabled) GHelper shows optimized GPU mode but it's still on iGPU + dGPU mode on battery. It does not switch to Eco automatically anymore, and it stays that way until I manually swap to Eco. How would I fix this? It was working normally a few weeks ago.

2022 G14 GA402RJ, Ryzen 9 6900HS, RX 6700s

**Comment by seerge** [comment]
@xzvs hello, I think a good start would be posting [an issue](https://github.com/seerge/g-helper/issues) and including a log from `%appdata%\GHelper` as asked in an issue template :)

But generally - on each start (and power source change) app would set last performance mode used on battery / plugged in and also enable / disable GPU if it's in Optimized .

**Reply by xzvs** [reply]
Alright I will post an issue later. Thank you!
